src/scripts.js

```diff
@@ -344,6 +340,21 @@ function combineDataSets(recipeData, ingredientsData) {
   })
 }
 
+function returnFilteredRecipes(recipes, search) {
+  let result = [];
+  result = recipes.filter(recipe => {
```

Comment:
```suggestion

  const result = recipes.filter(recipe => {
``` 
because filter returns an array, we could use const here